web/gridt/tests/unit/models/test_user.py

```diff
@@ -31,3 +35,20 @@ def test_hash(self):
     def test_avatar(self):
         user = User("username", "test@test.com", "test")
         self.assertEqual(user.get_email_hash(), "b642b4217b34b1e8d3bd915fc65c4452")
+
+    def test_get_password_reset_token(self):
+        with self.app_context():
+            user = User("username", "test@test.com", "password")
+            current_app.config["SECRET_KEY"] = "test"
```

Comment:
Why did you set this here?

Reply:
Not strictly necessary, if we assume that the secret key is set correctly.